packages/frontend/src/components/Modals/MarkerModal/MarkerModal.tsx

```diff
@@ -117,9 +128,10 @@ const MarkerModal: React.FC<MarkerModalProps> = ({ className, children, selected
     const disclaimerWithLink = useMemo(
         () => (
             <>
-                <a href="#" onClick={handleShare} className="invite-link">
+                {/* eslint-disable-next-line react/button-has-type */}
+                <button onClick={handleShare} className="invite-link">
```

Comment:
Having a button instead of an `a` will break the ui due to default styles being applied to the button tag:

<img width="814" alt="Bildschirmfoto 2024-11-28 um 16 15 33" src="https://github.com/user-attachments/assets/d565f8b5-6ea2-4a92-a71b-26e91c4b34d9">